README.md

```diff
@@ -106,7 +106,7 @@ If you experience a problem with this plugin you might be asked to collect langu
 
 ## Requirements
 <!-- Make sure to also update the note at the top of this file, and in user-guide.md -->
-Version 25.0.8 requires **IntelliJ IDEA 2024.2.\***, **2024.3.\***, or **2025.1.\*** and a minimum of **Java 17**.
+Version 25.0.9 requires **IntelliJ IDEA **2024.3.\***, **2025.1.\*** or **2025.2.\*** and a minimum of **Java 17**.
```

Comment:
`IntelliJ IDEA **2024` it is not clear to me what these markdown characters are going to do in the middle of a bolded section. Does this look ok?